### PR TITLE
Added the recordings that we are going to use for the run-android tests

### PR DESCRIPTION
**Pull request by digeff**
* Added the recordings that we are going to use for the run-android tests

The arguments, date, and events fields are generated automatically by the process execution recorder. All the other fields are created manually.

The idea of the metadata is so we know exactly what was the status of the "Machine/Environment" when we recorded this trace, so we know what kind of things might have affected them. At the moment we only use the events property in the tests, the remaining fields are for information purposes.

The .unused recordings are cases that I ran while trying to generate the traces I needed. They are not currently used by any test, but I store them because I don't know how to reproduce them, and they might be useful for future tests.

### CODE REVIEW

**Comment by hamiltonia** [comment]
:+1:

File: src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json [hunk 1]
```diff
@@ -0,0 +1,850 @@
+{
+    "title": "react-native run-android runs succesfully when ten VS Emulators are online",
+    "composedFrom": "succedsWithFiveVSEmulators",
```
Comment (dlebu) [inline src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json]: How is this `composedFrom` field used in tests?

Comment (digeff) [inline src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json]: It's not. All the metadata is informational at the moment.
I just wanted to say which was the original file that I modified to create this test, given that it's synthetic and not recorded.

Comment (meenakb) [inline src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json]: Nice. nit: typo in words related to success - **successfully, succeeds**

**Comment by meenakb** [comment]
:+1:

File: src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json (hunk 1 above)
Comment (digeff) [inline src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json]: That explains why I always fail :(
Fixing soon!

**Pull request merged**